src/components/TextCard.css

```
@@ -1,53 +1,64 @@
-}
+@layer liebe-base, liebe-theme, liebe-user;
 
-/* Responsive text sizing */
-@media (max-width: 640px) {
+@layer liebe-base {
+  /* TextCard specific styles */
   .text-card-content {
-    font-size: 0.9em;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)